sixtyfps_compiler/generator/rust.rs

```diff
@@ -214,7 +214,19 @@ fn handle_property_binding(
         let tokens_for_expression = compile_expression(binding_expression, component);
         init.push(if is_constant {
             let t = rust_type(&prop_type).unwrap_or(quote!(_));
-            quote! { #rust_property.set((||-> #t { (#tokens_for_expression) as #t })()); }
+
+            let mut uses_return = false;
```

Comment:
```suggestion
            // When there is a `return` statement, we must use a lambda expression in the generated code so that the 
            // generated code can have an actual return in it. We only want to do that if necessary because otherwise
            // this would slow down the rust compilation
            let mut uses_return = false;
```

I just think we can add a comment saying this is about the rust compilation, because otherwise one might "optimize" this code by reverting the patch again.  Feel free to change my wording.